### PR TITLE
add new G$ design system, and update to new screens for claim flow

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description

- Adds new values from G$ design system
- Refactor styles for claiming to match new screens

About # (link your issue here)
[#243](https://github.com/GoodDollar/GoodProtocolUI/issues/243)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: packages/good-design/src/core/layout/BalanceGD.tsx [hunk 1]
```diff
@@ -1,34 +1,53 @@
...
-  </View>
-));
+  </Box>
+);
+
+const BalanceView: FC<Required<BalanceGDProps> & { amount: CurrencyValue }> = memo(({ gdPrice, amount, chain }) => {
+  const network = chain === 122 ? "Fuse" : "Celo";
+  const copies = [
+    {
+      heading: "Your Balance",
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/BalanceGD.tsx:27]:
```suggestion
      id: "your-balance-label",
      heading: "Your Balance",
```

File: packages/good-design/src/core/layout/BalanceGD.tsx [hunk 2]
```diff
@@ -1,34 +1,53 @@
...
+      subheading: `on ${network}`
+    },
+    {
+      heading: amount.format({ suffix: "", prefix: amount.currency.ticker + " " }),
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/BalanceGD.tsx:31]:
```suggestion
      id: "your-balance-value",
      heading: amount.format({ suffix: "", prefix: amount.currency.ticker + " " }),
```

File: packages/good-design/src/core/layout/BalanceGD.tsx [hunk 3]
```diff
@@ -1,34 +1,53 @@
...
+      subheading: "(USD " + gdPrice.multiply(amount.format({ suffix: "", thousandSeparator: "" })).toFixed(2) + ")"
+    }
+  ];
+
+  return (
+    <View w="full" flexDirection="column" alignItems="center">
+      {copies.map(copy => (
+        <BalanceCopy key={copy.heading.charAt(0)} heading={copy.heading} subHeading={copy.subheading} />
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/BalanceGD.tsx:39]:
```suggestion
        <BalanceCopy key={id} heading={heading} subHeading={subheading} />
```

File: packages/good-design/src/core/layout/BalanceGD.tsx [hunk 4]
```diff
@@ -1,34 +1,53 @@
...
+    },
+    {
+      heading: amount.format({ suffix: "", prefix: amount.currency.ticker + " " }),
+      subheading: "(USD " + gdPrice.multiply(amount.format({ suffix: "", thousandSeparator: "" })).toFixed(2) + ")"
+    }
+  ];
+
+  return (
+    <View w="full" flexDirection="column" alignItems="center">
+      {copies.map(copy => (
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/BalanceGD.tsx:38]:
```suggestion
      {copies.map(({ id, heading, subheading }) => (
```

File: packages/good-design/src/core/layout/ClaimCard.tsx [hunk 5]
```diff
@@ -28,49 +28,52 @@ const ClaimCard: FC<ClaimCardProps> = ({ backgroundColor, titleColor, descriptio
...
           )}
 
           {!!contentItem.list && (
-            <View pt="30">
+            <View pt="30" textAlign="center">
               {contentItem.list?.map((item, index, list) => (
-                <Text key={index} color="dimgray" bold fontSize="16" fontFamily="subheading" fontWeight="semibold" pb={index === list.length - 1 ? "0" : "5"}>
-                  {item.key} <Text color="main">{item.value}</Text>
+                <Text
+                  key={index}
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/ClaimCard.tsx:65]: no index as key!

File: packages/good-design/src/core/layout/ClaimCarousel.tsx [hunk 6]
```diff
@@ -18,30 +19,18 @@ const SlideMark: FC<SlideMarkProps> = memo(({ isActive, isLast }) => (
...
-    .fill(0)
-    .map((_, index, arr) => (
-      <SlideMark key={index} isActive={index === activeSlide} isLast={index === arr.length - 1} />
-    ))}
+const SlidesComponent = memo(({ activeSlide, slidesNumber }: { activeSlide: number; slidesNumber: number }) => (
+  <>
+    {Array(slidesNumber)
+      .fill(0)
+      .map((_, index, arr) => (
+        <SlideMark key={index} isActive={index === activeSlide} isLast={index === arr.length - 1} />
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/ClaimCarousel.tsx:30]: no index as list key! unique value required

File: packages/good-design/src/theme/theme.ts [hunk 7]
```diff
@@ -26,26 +26,77 @@ export const theme = extendTheme({
...
   },
   components: {
     ...layout,
     ...buttons,
     ...advanced,
     Text: {
       baseStyle: {
-        fontFamily: 'body',
-        fontWeight: 'normal'
+        fontFamily: "body",
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/theme/theme.ts:82]: 1. create `theme/nativebase` directory
2. create `Text.theme.ts`

```typescript
export const Text = {
  baseStyle: {
    fontFamily: 'body',
    // ...
```
3. generate index via ctix
4. import from `./nativebase` and add to `components` prop

```typescript
import nativebase from './nativebase';

// ... 
components: {
  ...layout,
  ...buttons,
  ...advanced,
  ...nativebase,
```

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

File: packages/good-design/src/core/layout/ClaimCarousel.tsx [hunk 8]
```diff
@@ -27,7 +27,7 @@ const SlidesComponent = memo(({ activeSlide, slidesNumber }: { activeSlide: numb
     {Array(slidesNumber)
       .fill(0)
       .map((_, index, arr) => (
-        <SlideMark key={index} isActive={index === activeSlide} isLast={index === arr.length - 1} />
+        <SlideMark key={_} isActive={index === activeSlide} isLast={index === arr.length - 1} />
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/core/layout/ClaimCarousel.tsx:30]: number of slide is the same index,
some unique value describing slide should be used

for example "what-is-g$", "how-to-claim", "why-i-should-fv" etc 
e.g. **depending on slide contents**

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

File: packages/good-design/src/core/layout/ClaimCarousel.tsx (hunk 8 above)
Comment (L03TJ3) [reply packages/good-design/src/core/layout/ClaimCarousel.tsx]: done

File: packages/good-design/src/core/layout/ClaimCard.tsx (hunk 5 above)
Comment (L03TJ3) [reply packages/good-design/src/core/layout/ClaimCard.tsx]: done

File: packages/good-design/src/core/layout/ClaimCarousel.tsx (hunk 6 above)
Comment (L03TJ3) [reply packages/good-design/src/core/layout/ClaimCarousel.tsx]: done

File: packages/good-design/src/theme/theme.ts (hunk 7 above)
Comment (L03TJ3) [reply packages/good-design/src/theme/theme.ts]: done

**Review by johnsmith-gooddollar — APPROVED** [review]

**Pull request merged**